src/bindings/python/test.py

```diff
@@ -50,7 +51,43 @@
 
                 print("  %s %12s%12f" % (prefix, candidate['plate'], candidate['confidence']))
 
+finally:
+    if alpr:
+        alpr.unload()
+
+
+try:
+    alpr = Alpr(options.country, options.config, options.runtime_data)
```

Comment:
This is not right - it's not necessary to duplicate all of this code here (for example, this code would print out which version of OpenALPR is in use twice). Instead, there should be a new command line flag added to allow the program to run in OpenCV mode. Changes I suggest:

* add a new command-line argument `parser.add_argument("--cv", action="store_true", help="Enable OpenCV interpretation of the plate image")`
* remove the unconditional `import cv2` from the top of the file, and do a local import only if the flag is set.
* add an `if` statement around lines 32-33 and use an OpenCV alternative if the `cv` flag is set.

```
if options.cv:
    import cv2
    img = cv2.imread(options.plate_image)
    results = alpr.recognize_image(img)
else:
    jpeg_bytes = open(options.plate_image, "rb").read()
    results = alpr.recognize_array(jpeg_bytes)
```